### PR TITLE
Rename scripts to swap "-" for "_" for consistency

### PR DESCRIPTION
**Pull request by nyalldawson**
We had a confusing and unpredictable mix of script names with - vs _, e.g. `scripts/prepare-commit.sh` vs `scripts/sipify_all.sh`

This unifies with the preferred _ convention for script file names

### CODE REVIEW

**Comment by NathanW2** [comment]
+1

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Proper formatting
 Before making a pull request, please make sure your code is properly formatted
 by running the prepare commit script **before** issuing `git commit`.
 
-    ./scripts/prepare-commit.sh
+    ./scripts/prepare_commit.sh
 
 This can be automated by setting up the pre-commit hook properly.
 
-    ln -s ../../scripts/prepare-commit.sh .git/hooks/pre-commit
+    ln -s ../../scripts/prepare_commit.sh .git/hooks/pre-commit
```
Comment (m-kuhn) [inline .github/CONTRIBUTING.md:16]: This change will make it hard to work on different branches. I would be in favor of leaving a sym link of prepare-commit.sh in place for a transition period at least (i.e. until 3.10 is eol).

Comment (nyalldawson) [reply .github/CONTRIBUTING.md:16]: Good point, I've added the link

File: scripts/update_ts.sh [hunk 2]
```diff
@@ -194,7 +194,8 @@ EOF
 
 	QT_INSTALL_HEADERS=$(qmake -query QT_INSTALL_HEADERS)
 
-	echo "TR_EXCLUDE = ${QT_INSTALL_HEADERS%}/*" >>qgis_ts.pro
+	echo "TR_EXCLUDE = ${QT_INSTALL_HEADERS%
```
Comment (m-kuhn) [inline scripts/update_ts.sh:197]: Was this change on purpose? I can't really tell the purpose ...

Comment (nyalldawson) [reply scripts/update_ts.sh:197]: Nope, accident

Comment (nyalldawson) [reply scripts/update_ts.sh]: Fixed

**Comment by NathanW2** [comment]
Nice work.

>

**Pull request merged**